waves/src/Bobtimus.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from "react";
+import { SSEProvider } from "react-hooks-sse";
+import { CreateSwapPayload } from "./wasmProxy";
+
+export async function fundAddress(address: string): Promise<any> {
+    await fetch("/api/faucet/" + address, {
+        method: "POST",
+    });
+}
+
+export async function postSellPayload(payload: CreateSwapPayload) {
+    let res = await fetch("/api/swap/lbtc-lusdt/sell", {
```

Comment:
This route doesn't match with what bobtimus listens on? It also didn't match before but I just saw it now!

Reply:
Maybe I'm missing something, but I think it is correct: 
https://github.com/comit-network/droplet/blob/3c1b8e3cd804d45981b7fecd9d41615e516a1235/bobtimus/src/http.rs#L35-L40

Reply:
Ah my bad, I was looking at the `rate` endpoint :)